<launch>
  <node pkg="robot_self_filter" type="self_filter" respawn="true" name="laser_self_filter" output="screen">
     <remap from="cloud_in" to="/drop/points" />
     <remap from="cloud_out" to="/drop/points_self_filtered" />
     <param name="sensor_frame" type="string" value="/camera_rgb_optical_frame" />
     <param name="subsample_value" type="double" value=".02"/>
     <rosparam command="load" file="$(find pr2_arm_navigation_perception)/config/self_filter_right_left.yaml" />
  </node>
  
  <node pkg="lfd" type="exec_verb_traj_service.py" name="exec_verb_traj"/>
  <node pkg="lfd" type="make_verb_traj_service.py" name="make_verb_traj"/>
  <node pkg="jds_image_proc" name="interactive_segmentation" type="interactive_segmentation_service.py"
    args="--grabcut 0"
    output="screen"/>

</launch>